pkg/server/grpcServer.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"sixletters/simple-db/pkg/storage"
+	"sixletters/simple-db/pkg/tree"
+	"sixletters/simple-db/pkg/stubs"
+)
+
+type grpcServer struct{
+	storageEngine storage.StorageEngine
+}
+
+func NewGrpcServer() *grpcServer {
+	newStorageConfig := storage.NewConfigWithOpts(
```

Comment:
see how its done in the http server, you dont build a new storage engine everytime, its a singleton that is shared between all servers, not an instance that is instantiated everytime